src/components/DynamicTable/DynamicTable.tsx

```diff
@@ -28,10 +30,15 @@ export const DynamicTable = ({
   dataQa = null,
   noPadding = false,
   pagination,
+  minWidth,
   ...props
 }: DynamicTableProps) => {
   return (
-    <div className={classNames("styled-atlaskit-table", { "no-padding": noPadding }, className)} data-qa-id={dataQa}>
+    <div
+      className={classNames("styled-atlaskit-table", { "no-padding": noPadding }, className)}
+      data-qa-id={dataQa}
+      style={{ minWidth }}
```

Comment:
setting minWidth is a convenient way to maintain the gap between columns. setting it in css is not worth the hassle imo.